### PR TITLE
added condomless sex acts event

### PR DESCRIPTION
**Pull request by kaftand**
Added two events to facilitate PEP simulation.

MaritalCondomlessSexAct
NonMaritalCondomlessSexAct

Each event is triggered (max once per timestep per relationship) when an individual triggers a condomless sex act, stratified by marital status.

@Bridenbecker

### CODE REVIEW

File: utils/EventTrigger.cpp [hunk 1]
```diff
@@ -73,6 +73,9 @@ namespace Kernel
     const EventTrigger& EventTrigger::PFA_SeekingPartner             = EventTriggerFactory::CreateBuiltInEventTrigger( "PFA_SeekingPartner"             );
     const EventTrigger& EventTrigger::PFA_FoundPartner               = EventTriggerFactory::CreateBuiltInEventTrigger( "PFA_FoundPartner"               );
     const EventTrigger& EventTrigger::PFA_NoPartnerFound             = EventTriggerFactory::CreateBuiltInEventTrigger( "PFA_NoPartnerFound"             );
+    const EventTrigger & EventTrigger::MaritalCondomlessSexAct       = EventTriggerFactory::CreateBuiltInEventTrigger( "MaritalCondomlessSexAct"        );
+    const EventTrigger & EventTrigger::NonMaritalCondomlessSexAct    = EventTriggerFactory::CreateBuiltInEventTrigger( "NonMaritalCondomlessSexAct"     );
+
```
Comment (stitova-idm) [inline utils/EventTrigger.cpp:78]: nit: formatting

File: Regression/sanity.json [hunk 2]
```diff
@@ -1,13 +1,5 @@
...
-        { "path" : "Generic/30_Generic_Seattle_SingleCore" },
-        { "path" : "Vector/1_Vector_Sandbox" },
-        { "path" : "Vector/68_Vector_Genetics_OneGene_DiversityByRelease" },
-        { "path" : "Malaria/1_Malaria_Sandbox" },
-        { "path" : "Malaria/24_Malaria_SurveyReport_Multicore" },
-        { "path" : "Malaria/88_Malaria_FPG_TwoMarkers" },
-        { "path" : "STI/40_STI_Trivial_All_Reporters" },
-        { "path" : "STI/52_STI_All_Rel_Types_Assortivity_Prop_Location_Matrix" },
-        { "path" : "HIV/12_HIV_CascadeOfCare"}
+        { "path" : "HIV/45_HIV_CascadeOfCare_MultiNode_MultiCore"}
```
Comment (stitova-idm) [inline Regression/sanity.json:3]: should we keep the other tests in the sanity check? feels less sane with less checks?

Comment (Bridenbecker) [reply Regression/sanity.json:3]: Yes, I don't think we need this here.  hiv.json has this test in it and that is what is used.  sanity.json is meant to be a developer quick check that you didn't break anything big.

File: utils/EventTrigger.cpp (hunk 1 above)
Comment (Bridenbecker) [reply utils/EventTrigger.cpp:78]: @stitova-idm I don't understand your comment.  I think the formatting looks great.

**Review by Bridenbecker — APPROVED** [review]
@kaftand Thank you for doing this change.  I need to decide how we want to finish updating the tests.

**Comment by Bridenbecker** [comment]
Running
https://jenkins.apps.idmod.org/job/EMOD_Builds/job/scons_linux_hiv_regression/7/
https://jenkins.apps.idmod.org/job/EMOD_Builds/job/scons_linux_hiv_sfts/3/

File: utils/EventTrigger.cpp (hunk 1 above)
Comment (stitova-idm) [reply utils/EventTrigger.cpp:78]: image

Comment (Bridenbecker) [reply utils/EventTrigger.cpp:78]: didn't see that.  not going to hold up for that

**Pull request merged**